pkg/apis/config/defaults.go

```diff
@@ -44,6 +44,14 @@ const (
 	// DefaultMaxRevisionTimeoutSeconds will be set if MaxRevisionTimeoutSeconds is not specified.
 	DefaultMaxRevisionTimeoutSeconds = 10 * 60
 
+	// DefaultRevisionRequestStartTimeoutSeconds will be set if RevisionRequestStartTimeoutSeconds is not specified.
+	// for backward compatibility will keep default similar to DefaultRevisionTimeoutSeconds,
+	// should be revised in future releases.
+	DefaultRevisionRequestStartTimeoutSeconds = 5 * 60
```

Comment:
Let's match the property name

```suggestion
	DefaultRevisionResponseStartTimeoutSeconds = 5 * 60
```